server/task_executor_schema_update_ghost_cutover.go

```diff
@@ -21,6 +29,79 @@ type SchemaUpdateGhostCutoverTaskExecutor struct {
 	completed int32
 }
 
+func cutover(ctx context.Context, server *Server, task *api.Task, statement, schemaVersion string, vcsPushEvent *vcsPlugin.PushEvent, socketFilename, postponeFilename string) (terminated bool, result *api.TaskRunResultPayload, err error) {
```

Comment:
nit: put the most important information first. This means the callers should come before the callees within the same file.

I'm making a patch commit here.